docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
       Sitecore_Sitecore__IdentityServer__CertificateRawData: ${SITECORE_ID_CERTIFICATE}
       Sitecore_Sitecore__IdentityServer__PublicOrigin: https://${ID_HOST}
       Sitecore_Sitecore__IdentityServer__CertificateRawDataPassword: ${SITECORE_ID_CERTIFICATE_PASSWORD}
-      Sitecore_License: ${SITECORE_LICENSE}
+      # Sitecore_License: ${SITECORE_LICENSE}
```

Comment:
For the build to succeed, could you please apply [the same change as this PR](https://github.com/Sitecore/Sitecore.Demo.XmCloud.PlaySummit/pull/38/files#diff-95c3206620a390a5bf07d14555adddce039983ba4369bd3b1bee7c002986f380) in the `docker/build-and-push-images.yml` file?

Aka change `$(sitecore.container.registry.password)` to `${env:SITECORE_CONTAINER_REGISTRY_PASSWORD}`

Thanks a lot!